x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/create_security_rule_type_wrapper.ts

```diff
@@ -519,6 +520,21 @@ export const createSecurityRuleTypeWrapper: CreateSecurityRuleTypeWrapper =
                   },
                 });
               }
+
+              const disabledAction = rule.actions.find(
+                (action) => !actions.isActionTypeEnabled(action.actionTypeId)
+              );
+              if (disabledAction) {
+                await ruleExecutionLogger.logStatusChange({
+                  newStatus: RuleExecutionStatusEnum['partial failure'],
+                  message: `Alerts created but the registered action ${disabledAction.actionTypeId} is disabled. Please check your license / tier and ensure the connector is enabled for your given license / tier`,
+                  metrics: {
+                    searchDurations: result.searchAfterTimes,
+                    indexingDurations: result.bulkCreateTimes,
+                    enrichmentDurations: result.enrichmentTimes,
+                  },
+                });
+              }
```

Comment:
Rule execution had multiple warnings: 2 for disabled actions, another for number of alerts exceeding max_signal.
But max_signals warning is lost

Reply:
Ah thanks. This should be fixed with the latest commit along with some other changes to ensure it is written at correct intervals of the rule execution.

<img width="1192" alt="max_alerts_disabled_action" src="https://github.com/elastic/kibana/assets/915763/f80ec857-6280-42cb-8305-98dc90c81669">

Thinking out loud here, I was looking at how we join all the warning / error messages into a single `RuleStatusFailedCallOut`. Maybe we could look into displaying a smaller callout at the top with links directly to the rule execution table on the rule details page rather? I feel like it would clean up the UI a bit and we could better replicate that logic in the security rule type wrapper so that we only have to write all the warnings at one time rather than throughout different places in the code. I'll open an issue but figured I'd post here as well.